.github/ISSUE_TEMPLATE/map_report.yml

```
@@ -0,0 +1,29 @@
+name: "Сообщить о проблеме на карте"
+title: "[Название карты или локации] "
+description: "Если что-то не так с картой"
+labels: ["Bug", "Map Issue"]
+body:
+  - type: textarea
+    id: location
+    validations:
+      required: true
+    attributes:
+      label: "В какой локации это произошло?"
+      description: "Укажите станцию или локацию на которой вы нашли мапперскую ошибку"
+      placeholder: |
+        Исход
+        Гамма
+        Астероид
+        Астероид Прометея
+        Дирелект с базой советов
+        База Синдиката
+
+  - type: textarea
+    id: description
+    validations:
+      required: true
+    attributes:
+      label: "Опишите, что по вашему не так на карте?"
```

```suggestion
      label: "Описание"
```
В description: оно повторяется